src/pyhf/infer/mle.py

```diff
@@ -47,6 +47,14 @@ def twice_nll(pars, data, pdf):
     return -2 * pdf.logpdf(pars, data)
 
 
+def _validate_fit_inputs(pdf, init, bounds, fixed_params):
```

Comment:
the `pdf` is not needed is it?

Reply:
not right now, (I first had it check only the POI for which I needed `pdf.config`. will remove